docs/compression_algorithms/post_training/Quantization.md

```diff
@@ -0,0 +1,88 @@
+## Post-Training Quantization
+
+Post-Training Quantization is a quantization algorithm that doesn't demand retraining of a quantized model.
+It utilizes a small subset of the initial dataset to calibrate quantization constants.
+
+NNCF provides an advanced Post-Training Quantization algorithm, which consists of the following techniques:
+
+1) MinMaxQuantization - Analyzes the model and inserts extra quantization layers calibrated on a small subset.
+2) FastBiasCorrection or BiasCorrection - Reduces the bias errors between quantized layers and the corresponding
+   original layers.
+
+### Usage
+
+To start the algorithm, provide the following entities:
+
+* Original model.
+* Validation part of the dataset.
+* [Data transformation function](#data-transformation-function) from the original dataset format to the NNCF format.
+
+The basic workflow steps:
+
+1) Create the [data transformation function](#data-transformation-function).
+
+```python
+def transform_fn(data_item):
+    images, _ = data_item
+    return images
```

Comment:
It is confusing. When you see it for the first time probably you just copy-paste this code to your example. It may not be suitable for your case.

I think it is better to write here something like this:

```
# Transform `data_item` to the model's input data.
```

Reply:
This suggestion was from @AlexKoff88 to have an initial understanding what it is behind these steps. I think that this is a good suggestion.

Reply:
I think we should align here with other steps. So, they either should provide code snippets or not.